gapis/api/vulkan/replay.go

```diff
@@ -1026,3 +1051,25 @@ func (a API) QueryIssues(
 	}
 	return res.([]replay.Issue), nil
 }
+
+func (a API) QueryTimestamps(
+	ctx context.Context,
+	intent replay.Intent,
+	mgr replay.Manager,
+	hints *service.UsageHints) ([]replay.Timestamps, error) {
+
+	_, err := resolve.SyncData(ctx, intent.Capture)
```

Comment:
If we are not using the syncData, you can avoid resolving it here. If we never end up needing it, then we can save the time ever generating it.

Reply:
removed